gpustack/utils/platform.py

```diff
@@ -60,20 +67,24 @@ def is_inside_kubernetes() -> bool:
 
 
 def is_supported_higress(kubeconfig: Optional[str] = None) -> bool:
-    api_client = None
+    configuration = Configuration()
     if kubeconfig and os.path.exists(kubeconfig):
-        api_client = config.new_client_from_config(config_file=kubeconfig)
+        cfg_loader = KubeConfigLoader(config_dict=KubeConfigMerger(kubeconfig).config)
+        if not cfg_loader._load_user_token():
+            cfg_loader._load_user_pass_token()
+        cfg_loader._load_cluster_info()
+        cfg_loader._set_config(configuration)
+    elif is_inside_kubernetes():
+        cfg_loader = InClusterConfigLoader(
+            token_filename=SERVICE_TOKEN_FILENAME,
+            cert_filename=SERVICE_CERT_FILENAME,
+        )
+        cfg_loader.load_and_set(configuration)
```

Comment:
Is there any special consideration here? Why not use `kubernetes.config.load_config()`?